src/main/java/graphql/GraphQL.java

```diff
@@ -244,50 +244,108 @@ public ExecutionResult execute(String requestString, String operationName, Objec
      * @return result including errors
      */
     public ExecutionResult execute(ExecutionInput executionInput) {
```

Comment:
I would rather see a new PR with the GraphQL class refactorings say then all the new Builder() changes